content/implementation-path.md

```diff
@@ -0,0 +1,127 @@
+
+### Add concurrent support 
+
+Where practical add support for new names while supporting old names. This allows projects to change incrementally instead of requiring changes to occur simultaneously across all related projects and organizations. It also reduces the risk of system failure.
+
+For projects that have automated test suites, add tests to ensure that either both old and new names work, or that at a minimum new names work. This allows end users to incrementally transition with confidence.
+
+In some projects this work will take significant time, be gracious to those doing the work, and be understanding that it will take time to do all this coordination.
+ 
+## Customer-facing changes
+
+Changes to customer facing terminology like URLs are harder to deal with. Run through the steps for internal changes in addition to the following two points.
+
+### URLs, URL fragments, command and API names
+
+For HTML URLs like documentation, add new `id` values, or rename the page and set up a server 301 redirect from the old URL to the new one.
+
+Where possible, API endpoints should accept both terms for a period of time, and eventually deprecate the old term.
+
+Similarly, command names and flags for CLI tooling should accept both terms for a period of time. Ideally, serve a deprecation notice when a user uses the old command or flag.
+
+
+### Assess downstream impact
+
+Assess whether downstream projects have a specific dependency on any of the following:
+
+* Git branch names
+* Specific URLs
+* Specific keywords or commands
+
+Catalog downstream tools which have a specific dependency on git branch names. Any tool that needs to know the specific URL of a git repository should be checked for references to the branch name.
+
+If the work is done well, projects which consume open source software won’t face any immediate impact: old API endpoints, method names, URLs and more will continue to work in the short-term – about 1-3 years, depending on the project. 
+
+However, in the long term, if downstream projects keep their dependencies up to date, they will need to change calls to methods, endpoints, and other exposed functionality. URLs will likely maintain aliases in the long term.
+
+## Project tooling
+
+Review all systems that build or test systems. This includes:
+
+* any continuous integration / continuous delivery (CI/CD) systems
+* git branch names
+* GitHub Actions
+* Static site generator CDNs like Netlify 
+
+Check their external configuration settings, especially all that are not controlled by the version control system for the source code.
+
+Review anything that has branch-specific information, such as protected branch rules
+
+### Changing the git branch name
+
+The best resource for changing branch names is [GitHub's renaming repository](https://github.com/github/renaming).
+
+Their recommendation is to wait until later in the year (2020) to change branch names. However, if you want to start work, we recommend the following:
```

Comment:
it is the end of 2020, has this been updated?

Reply:
It has. GitHub's official line is still end of 2020.